examples/clerk-nextjs/pages/schema.tsx

```diff
@@ -0,0 +1,37 @@
+import { useAuth } from '@clerk/nextjs'
+import useSWR from 'swr'
+
+export const useQuery = (query: any, variables?: any) => {
+  if (!query) {
+    throw Error('No query provided to `useQuery`')
+  }
+
+  const { getToken } = useAuth()
+
+  const fetcher = async () =>
+    await fetch(process.env.NEXT_PUBLIC_GRAFBASE_URL as string, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+        authorization: `Bearer ${await getToken({
```

Comment:
```suggestion
        'x-api-key': `${await getToken({
```

Reply:
Is this right? I was under the impression `x-api-key` was for API keys made on Grafbase?